documentation/rosetta-modelling-component.md

```diff
@@ -1530,23 +1540,25 @@ Mappings are expected to be one-to-one with each input value mapping to one Rose
 
 The `meta` keyword inside a synonym is used to map [metadata](#meta-data-annotation).
 
-In the below example, the value of the "issuer" input will be mapped to the value of the `issuer` attribute and the value of the "issuerIdScheme" metadata associated to "issuer" will be mapped to the `scheme` metadata attribute.
+In the below example, the value of the "combustible" input will be mapped to the value of the `fuelType` attribute and the value of the "scheme" metadata associated to "combustible" will be mapped to the `scheme` metadata attribute.
 
 ```
-issuer string (0..1)
-  [metadata scheme]
-  [synonym FpML_5_10 value "issuer" meta "issuerIdScheme"]
+type EngineSpecification:
```

Comment:
You are using the word "scheme" twice. To avoid confusion, can't you have 
[synonym META_SCHEME_EXAMPLE_1 value "combustible" meta "fuelTypeScheme"]?

Reply:
yes I see that "scheme" could be confussing, I will change to proposed "fuelTypeScheme" both in translate-demo and here in the documentation